packages/block-library/src/separator/theme.scss

```diff
@@ -1,9 +1,15 @@
+// Import styles for rendering the static content of deprecated block versions.
+@import "./deprecated.scss";
+
 .wp-block-separator {
 	border: none;
 	border-bottom: 2px solid currentColor;
 	margin-left: auto;
 	margin-right: auto;
-	opacity: 0.4;
+
+	&.has-alpha-channel-opacity {
```

Comment:
@ramonjd do you think we should maybe change this to `&:where( .has-alpha-channel-opacity )` so easier for themes to override if needed?

Reply:
This sounds appealing but I think it lowers the specificity so that the styles from `deprecated.scss`, i.e., `.wp-block-separator ` and therefore `opacity: 0.4;` take precedence.


<img width="392" alt="Screen Shot 2022-03-21 at 1 53 40 pm" src="https://user-images.githubusercontent.com/6458278/159199381-4d9263c4-9135-4568-b7dc-b2e09e340d76.png">

Reply:
Actually, now I look again, I'm not sure where that `opacity: 0.4;` is coming from.

I'm looking at a new Separator using this branch

`<hr class="wp-block-separator has-alpha-channel-opacity is-style-wide">`

Yet the scss is

```scss
.wp-block-separator {
	// V1 version of the block expects a default opactiy of 0.4 to be set.
	&.has-css-opacity {
		opacity: 0.4;
	}
}
```